src/index.js

```diff
@@ -6,11 +6,22 @@ import { Provider } from "react-redux";
 import thunk from "redux-thunk";
 import { composeWithDevTools } from "redux-devtools-extension";
 import SplashScreen from "react-native-splash-screen";
+import Config from "react-native-config";
+import { Client, Configuration } from "bugsnag-react-native";
+
 import reducers from "./reducers";
 import { getEvents } from "./actions/events";
 import { loadSavedEvents } from "./actions/saved-events";
 import App from "./App";
 
+const bugsnagConfiguration = new Configuration();
+bugsnagConfiguration.releaseStage = Config.RELEASE_STAGE
+  ? Config.RELEASE_STAGE
+  : "dev";
+// Only send reports for releases from master branch
+bugsnagConfiguration.notifyReleaseStages = ["beta", "release"];
+const bugsnag = new Client(bugsnagConfiguration); // eslint-disable-line no-unused-vars
```

Comment:
Do we need to do anything with this client after we created it?

Reply:
Not for unhandled errors https://docs.bugsnag.com/platforms/react-native/

Eventually we should probably have some handled errors around contentful fetching